src/challenges/Day9.java

```diff
@@ -0,0 +1,150 @@
+package challenges;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.IntStream;
+
+public class Day9 {
+    public static void main(String[] args) {
+        int[][] heightmap = readFile();
+
+        List<Coordinate> lowPoints = getLowPoints(heightmap);
+        Map<Coordinate, Integer> basins = getBasins(heightmap, lowPoints);
+        int largestBasinSizesMultiplied = getLargestBasinSizesMultiplied(basins);
+
+        // System.out.println(sumRiskLevels(heightmap, lowPoints));
+        System.out.println(largestBasinSizesMultiplied);
+    }
+
+    private static int[][] readFile() {
```

Comment:
I'm at a loss for why it took me so long to separate reading into its own method